sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClientOptions.cs

```diff
@@ -37,7 +37,12 @@ public enum ServiceVersion
             /// <summary>
             /// Version 3.2-preview.1
             /// </summary>
-            V3_2_Preview_1 = 3
+            V3_2_Preview_1 = 3,
+
+            /// <summary>
+            /// Version 3.2-preview.2
+            /// </summary>
+            V3_2_Preview_2 = 4
```

Comment:
Usually we just support the GA versions + the latest preview, so you can remove the `V3_2_Preview_1` member and keep only `V3_2_Preview_2 = 3`.